tests/AutoReview/CiConfigurationTest.php

```diff
@@ -119,7 +119,7 @@ private static function ensureTraversableContainsIdenticalIsAvailable(): void
      */
     private static function assertUpcomingPhpVersionIsCoveredByCiJob(string $lastSupportedVersion, array $ciVersions): void
     {
-        if ('8.2' === $lastSupportedVersion) {
+        if ('8.4' === $lastSupportedVersion) {
```

Comment:
we already forgot to update it once (to 8.3)

let's check if it;s really needed to have this line
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8303

Reply:
We didn't bump it because I believe we already had 8.3 in the CI (2 jobs: tests + fixer) when the check was introduced, so it was passing. Maybe the check should be more rigorous?